README.md

```diff
@@ -0,0 +1,70 @@
+To get you started you can simply clone the repository
+
+git clone https://github.com/armine94/backend
+
+## Prerequisites
+you need `git` to `clone` the repository. You can get git from [http://git-scm.com/].
+you need `node`, you can `download` here [https://nodejs.org/en/download/]
+you need `npm`, you can `install`  npm install npm@latest -g
+
+A number of node.js tools is necessary to initialize the project. You must have node.js and its package manager (npm) installed. You can get them from http://nodejs.org/. The tools/modules used in this project are listed in package.json and include express, mongodb and mongoose.
+
+and install the dependencies
+npm ci
+
+## MongoDB
+
+The project uses MongoDB as a database.
+you need `mongodb`, you can `download` here  [https://docs.mongodb.com/manual/installation/]
+
+about what starts mongo  `https://docs.mongodb.com/manual/tutorial/manage-mongodb-processes/`
+
+## Run the Application
+
+1) for development
+npm run dev
+
+2) for production
+npm start 
+
+The project is preconfigured with a simple development web server. The simplest way to start this server is:
+
+## Project structure
+├── backend
+|  ├── configs -  Contains application configuration settings such as database configuration
+|  |  ├──  log.config.js
```

Comment:
no need a lot of small configs:
just use one envSettings.json with structure like this:
{
    server: {
       port: 3000 
   },
   db: {
      DB: 'mongodb://',
      IP: 'localhost',
      ...
   }
   ...
}